src/test/java/se/sundsvall/notes/api/NoteResourceTest.java

```diff
@@ -228,6 +229,45 @@ void findNotes() {
 		assertThat(findNotesRequest.getMunicipalityId()).isEqualTo(MUNICIPALITY_ID);
 	}
 
+	@Test
+	void getDifference() {
+		// Parameter values
+		final var id = UUID.randomUUID().toString();
+		final var from = 1;
+		final var to = 2;
+
+
+		final var response = webTestClient.get().uri(builder -> builder.path(PATH + "/{id}/difference").queryParam("from", 1).queryParam("to", 2).build(Map.of("id", id)))
+			.exchange()
+			.expectStatus().isOk()
+			.expectHeader().contentType(APPLICATION_JSON)
+			.expectBody(DifferenceResponse.class)
+			.returnResult()
+			.getResponseBody();
+
+		// Verification
+		assertThat(response).isNotNull();
+		//TODO: Add verification for response
+	}
+
+	@Test
+	void getDifferenceNoParameters() {
```

Comment:
Shouldn't this throw a 400 Bad Request?

Reply:
I think it will make it complicated to understand the API for the clients. What will happen if they only leave out "from" but provides "to", if they leave out "to" but provides "from"? This interpretation logic must be well documented if we choose to go down this road. Or, we can keep the API easy to understand ("to" and "from" must always be present) along with providing the clients with a list of revisions for a Note entity.